spec/models/validations/sales/property_validations_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+RSpec.describe Validations::Sales::PropertyValidations do
+  subject(:property_validator) { validator_class.new }
+
+  let(:validator_class) { Class.new { include Validations::Sales::PropertyValidations } }
+
+  describe "#validate_property_unit_type" do
+    context "when number of bedrooms is <= 1" do
+      let(:record) { FactoryBot.build(:sales_log, beds: 1, proptype: 2) }
+
+      it "does not add an error if it's a bedsit" do
+        property_validator.validate_property_unit_type(record)
+
+        expect(record.errors).not_to be_present
+      end
+    end
+
+    context "when number of bedrooms is > 1" do
+      let(:record) { FactoryBot.build(:sales_log, beds: 2, proptype: 2) }
+
+      it "does add an error if it's a bedsit" do
+        property_validator.validate_property_unit_type(record)
+
+        expect(record.errors).to be_present
+      end
+    end
+  end
+
+  describe "#validate_property_number_of_bedrooms" do
```

Comment:
Would it be simpler to combine this test and the above (so the following 2 lines would be sequential)?
`property_validator.validate_property_unit_type(record)`
`property_validator.validate_property_number_of_bedrooms(record)`